spec/features/internship_pages_spec.rb

```diff
@@ -0,0 +1,144 @@
+feature 'index page' do
+
+  context "as a student" do
+    let(:student) { FactoryGirl.create(:student) }
+    let!(:internship) { FactoryGirl.create(:internship, cohort_id: student.cohort.id) }
+    before { login_as(student, scope: :student) }
+
+    scenario 'students can see internships listed by comapany name' do
+      visit root_path
+      click_link 'Internships'
+      expect(page).to have_content internship.company_name
+    end
+
+    scenario 'students cannot see the edit or delete links' do
+      visit cohort_internships_path(student.cohort)
+      expect(page).to_not have_content 'Edit'
+    end
+
+    scenario 'students cannot see the new internship link' do
+      visit cohort_internships_path(student.cohort)
+      expect(page).to_not have_content '+ New Internship'
+    end
+  end
+
+  context "as an admin" do
+    let(:admin) { FactoryGirl.create(:admin) }
+    let!(:internship) { FactoryGirl.create(:internship, cohort_id: admin.current_cohort_id) }
```

Comment:
Without it, since let uses lazy evaluation, it won't be preloaded into memory when the the test on line 30 navigates to the internship page, causing the test to fail.